qa/suites/upgrade/quincy-x/parallel/1-tasks.yaml

```diff
@@ -0,0 +1,29 @@
+tasks:
+- install:
+    branch: quincy
+    exclude_packages:
+      - ceph-volume
+- print: "**** done install task..."
+- print: "**** done start installing quincy cephadm ..."
+- cephadm:
+    image: docker.io/ceph/daemon-base:latest-quincy
+    cephadm_branch: quincy
+    cephadm_git_url: https://github.com/ceph/ceph
```

Comment:
@adk3798 can you take a look at this?

Reply:
image should probably be `quay.io/ceph/daemon:latest-quincy-devel` instead

Reply:
testing on this fix:

https://pulpito.ceph.com/teuthology-2022-03-17_17:37:21-upgrade:quincy-x:parallel-master-distro-default-smithi

https://pulpito.ceph.com/teuthology-2022-03-17_17:38:06-upgrade:quincy-x:stress-split-master-distro-default-smithi

Reply:
http://pulpito.front.sepia.ceph.com/teuthology-2022-03-17_19:56:51-upgrade:quincy-x:parallel-master-distro-default-smithi
http://pulpito.front.sepia.ceph.com/teuthology-2022-03-17_18:23:54-upgrade:quincy-x:stress-split-master-distro-default-smithi

per @neha-ojha these tests show:

"http://pulpito.front.sepia.ceph.com/teuthology-2022-03-17_18:23:54-upgrade:quincy-x:stress-split-master-distro-default-smithi"

I think this is resolved and we will wait for a little before merging it with hope we will have the initial `reel` release additions